bayeux.go

```diff
@@ -237,13 +237,7 @@ func (b *Bayeux) connect(out chan TriggerEvent) chan TriggerEvent {
 	}()
 	return out
 }
-
-func GetSalesforceCredentials() Credentials {
-	clientID := mustGetEnv("SALESFORCE_CONSUMER_KEY")
-	clientSecret := mustGetEnv("SALESFORCE_CONSUMER_SECRET")
-	username := mustGetEnv("SALESFORCE_USER")
-	password := mustGetEnv("SALESFORCE_PASSWORD")
-	tokenURL := mustGetEnv("SALESFORCE_TOKEN_URL")
+func GetSalesforceCredentials(clientID string, clientSecret string, username string, password string, tokenURL string) Credentials {
```

Comment:
Let's organize them in the `AuthenticationParameters` structure. This way you can break more fields to Parameters and don't break any existing API :)